addons/website_blog/tests/test_website_blog_flow.py

```diff
@@ -171,3 +187,37 @@ def test_teaser_manual(self):
         self.assertEqual(blog_post_parseltongue.teaser, "New Parseltongue Content...", "Should still fallback to content")
         self.assertEqual(self.test_blog_post.teaser_manual, "English Teaser Manual")
         self.assertFalse(blog_post_parseltongue.teaser_manual, "Should still be empty")
+
+    def test_update_field_translation(self):
+        """Checks if translated text could be updated"""
+        self.authenticate('admin', 'admin')
+
+        # Setup
+        br_lang = self.env['res.lang']._activate_lang('pt_BR')
+        en_lang = self.env['res.lang']._activate_lang('en_US')
+
+        website = self.env['website'].browse(1)
+        website.language_ids += br_lang
+        website.default_lang_id = br_lang
+        blog_post = self.env['blog.post'].with_context(lang=br_lang.code).create({
+            'name':'Test Blog',
+            'content':'Todos os blogs', 
+            'website_id':website.id,
```

Comment:
`'website_id':website.id,` is not necessary for this test.
I am not sure if it is common that blog_post is always bound with a website, if so, leave it is OK